content/sensu-core/0.29/guides/pre-compile-plugins.md

```diff
@@ -44,27 +39,28 @@ gpgcheck=0
 enabled=1' | sudo tee /etc/yum.repos.d/sensu.repo && yum install -y sensu
 ```
 
-2. Install EPEL and Dev tools
+And then install EPEL and Dev tools
 
 ```bash
 sudo yum install -y epel-release
 sudo yum groupinstall -y "Development Tools"
 ```
-3. Add Sensu's embedded bin to our path
 
- ```bash
- export PATH=$PATH:/opt/sensu/embedded/bin
- ```
+Finally we'll add Sensu's embedded bin to our path
+
+```bash
+export PATH=$PATH:/opt/sensu/embedded/bin
+```
 
 ## Installing a Gem to a tmp directory
 
-1. Install gem-compiler
+Install the gem-compiler package
 
 ```bash
 gem install gem-compiler
```

Comment:
I get a permission error here, but running as `sudo` it can't find `gem` as `sudo` isn't aware of the previous `export` command. Becoming root, it works. I am not sure how to resolve this.